code/README.md

```diff
@@ -1,131 +1,221 @@
+ 
+### Get the code
+
+The IAMF library source code is stored in the Alliance for Open Media Git
+repository:
+
+~~~
+    $ git clone https://github.com/AOMediaCodec/libiamf
+    # By default, the above command stores the source in the libiamf/code directory:
+    $ cd libiamf/code
+~~~
+
+### Basic build
+
+"build.sh" is an example to build, you can run it directly at your side.  
+(dependent codec libraries and external libriaries complied under x64 linux have been provided in advance)
+
+CMake replaces the configure step typical of many projects. Running CMake will
+produce configuration and build files for the currently selected CMake
+generator. For most systems the default generator is Unix Makefiles. The basic
+form of a makefile build is the following:
+
+~~~
+    $ cmake .
+    $ make
+~~~
+
+### Configuration options
+
+The IAMF library has few configuration options, There are one option which is used to enable binaural rendering:
+    Build binaural rendering configuration options. These have the form `BINAURALIZER`.
+~~~
+    $ cmake ./ -DBINAURALIZER=ON
+    $ make
+~~~
+
+### Dylib builds
+
+A dylib (shared object) build of the IAMF library is enabled by default.
+~~~
+    $ cmake .
+    $ make
+~~~
+
+### Cross compiling
+
+For the purposes of building the IAMF and applications and relative to the
+scope of this guide, all builds for architectures differing from the native host
+architecture will be considered cross compiles. The IAMF CMake build handles
+cross compiling via the use of toolchain files included in the AV1 repository.
+The toolchain files available at the time of this writing are:
+
+- x86_64-mingw-gcc.cmake
+
+NOTE: Only x86_64 mingw cross compiling is available currently, in a future version we can consider adding more.  
+Please note that all dependent codec and external libraries should have been cross compiling already.
+
+The following example demonstrates use of the x86_64-mingw-gcc.cmake toolchain file on
+a x86\_64 Windows host:
+
+~~~
+    $ cmake ./ -DCMAKE_TOOLCHAIN_FILE=build/cmake/toolchains/x86_64-mingw-gcc.cmake
+    $ make
+~~~
+
+### Microsoft Visual Studio builds
+
+Building the IAMF library in Microsoft Visual Studio is supported. Visual
+Studio 2015 (14.0) solution has been provided.
+
+Open win32/VS2015/iamf.sln directly.
+
+## Testing the IAMF
+
+### 1. Build application
+
+iamfplayer is is a test application used to decode IAMF bitstream or mp4 file which is IAMF encapsulation.
+~~~
+    $ cd test/tools/iamfplayer
+
+    $ cmake -DCMAKE_INSTALL_PREFIX=${BUILD_LIBS} .
+    # ${BUILD_LIBS} is the iamf library and header files installing directory.
+    # If enable binaural rendering, add option `BINAURALIZER`
+
+    $ make
+~~~
+
+### 2. Decoder tests
+
+If want to run binaural cases, please download [default.tf](https://github.com/ebu/bear/releases/download/v0.0.1-pre/default.tf) file into your executing directory.
```

Comment:
How about replacing the line by:
To produce binaural output, please download the following file and place it in your working directory [default.tf](https://github.com/ebu/bear/releases/download/v0.0.1-pre/default.tf).